docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -238,6 +240,106 @@ per output type, which enables loadbalancing.
 |Supported
 |===
 
+[discrete]
+[[additional-capabilties-beats-and-agent]]
+== Additional capabilities
+
+The following table shows the additional capabilties supported by Beats and the {agent} in {version}:
+
+
+[options,header]
+|===
+|Item |Beats |{fleet}-managed {agent} |Standalone {agent}
+
+|Single agent for all use cases
+|{n}
+|{y}
+|{y}
+
+|Install integrations from Kibana UI or API
+|{n}
+|{y}
+|{y}
+
+|Configure from integrations UI
```

Comment:
```suggestion
|Configure from Integrations UI
```

Reply:
It's correct for integrations to be lowercase except when you are describing the UI.